.github/PULL_REQUEST_TEMPLATE/APOLLO_RELEASE_TEMPLATE.md

```diff
@@ -1,6 +1,4 @@
-Release @apollo/PACKAGE1@X.Y.Z [, @apollo/PACKAGE2@X.Y.Z]
-
-As with [release PRs in the past](https://github.com/apollographql/federation/issues?q=label%3A%22:label:%20release%22+is%3Aclosed), this is a PR tracking a `release-x.y.z` branch for an upcoming release. 🙌   The version in the title of this PR should correspond to the appropriate branch.
```

Comment:
I appreciate the extra context 🙏 

I think it makes sense to add a `gh` command in the instructions. I'll likely keep the web-page link version as well, since I'd like to keep the deployer required software to a minimum to encourage more people to create releases.

Reply:
Added in https://github.com/apollographql/federation/pull/1614/commits/15288bfcd7d4e06f8466a8edc71d331e35c639e8. Lemme know what you think